ui/components/src/LineChart/LineChart.tsx

```diff
@@ -112,6 +130,24 @@ export function LineChart({
   const [isDragging, setIsDragging] = useState(false);
   const [startX, setStartX] = useState(0);
 
+  useImperativeHandle(
+    ref,
+    () => {
+      return {
+        focusSeries({ id }: LineChartHandleFocusOpts) {
```

Comment:
Any thoughts on the naming for this? I was trying to communicate that it highlights *just* the specified series, so I used "focus" but maybe `highlightSeries` would implicitly include that? Let me know what you think!